src/Context/SpanContext.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace OpenTelemetry\Context;
+
+class SpanContext extends \OpenTelemetry\Tracing\SpanContext
```

Comment:
I think you were renaming Tracing to Trace.  Did you mean to change it here too?

Reply:
I followed instructions from here: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-layout.md and I want to rename Tracing to Trace.
For now, I just extended the previous implementation to make sure that it's ok for you to rename. After we decide this is the right path, I will move the implementation here.

For this case, I moved SpanContext in OpenTelemtry\Context.

Reply:
Yeah; following the spec here is perfect.  I don't believe this existed when we implemented originally (or was inadvertent).  This change sounds good.